### DISCUSSION TITLE
JSONDecodeError: Expecting value: line 1 column 1 (char 0)

### DISCUSSION THREAD

**Discussion by 1r00t** [vladmandic/sdnext#1139] [General]
Hey, I just came across this error when starting webui.sh.
I figured it was an empty `cache.json` and `cache.json.lock` file in the root folder. I deleted it and everything is fine now.
No idea what happened to that file, I did not edit it.

I'm creating this discussion so other people see it if they get the same error.

If you get `JSONDecodeError` just delete these two files :P

![grafik](https://github.com/vladmandic/automatic/assets/9479785/41d9abef-765e-4791-b82f-087724454276)

**Comment by vladmandic** [comment]
i'll add exception handler around that just in case, will be in the next version
